scripts/installer-lang/id.ps1

```diff
@@ -0,0 +1,66 @@
+    MsSpoti2        = "Copot pemasangan Spotify edisi Windows Store [Y/N]"
+    MsSpoti3        = "Menghapus instalan Spotify MS secara otomatis..."
+    MsSpoti4        = "Menghapus Instalasi Spotify MS..."
+    Prem            = "Modifikasi untuk akun premium..."
+    OldV            = "Menemukan versi lama dari Spotify"
+    OldV2           = "Spotify versi {0} sudah usang, disarankan untuk meningkatkan ke {1}"
+    OldV3           = "Ingin memperbarui ? [Y/N]"
+    AutoUpd         = "Pembaruan otomatis ke versi yang direkomendasikan"
+    DelOrOver       = "Apakah anda ingin mencopot pemasangan versi {0} saat ini atau menimpanya? Y [Copot Pemasangan] / N [Timpa]"
+    DelOld          = "Menghapus instalasi Spotify lama..."
+    NewV            = "Versi Spotify yang tidak didukung ditemukan"
+    NewV2           = "Spotify versi {0} belum diuji, saat ini yang stabil adalah versi {1}"
+    NewV3           = "Apakah anda ingin melanjutkan dengan versi {0} (Memungkinkan terjadinya kesalahan)? [Y/T]"
+    Recom           = "Apakah anda ingin memasang versi {0} yang disarankan? [Y/T]"
+    DelNew          = "Menghapus instalasi Spotify yang belum teruji..."
+    DownSpoti       = "Mengunduh dan menginstal Spotify"
+    DownSpoti2      = "Mohon ditunggu..."
+    PodcatsOff      = "Podcast mati"
+    PodcastsOn      = "Podcast hidup"
+    PodcatsSelect   = "Apakah anda ingin menonaktifkan podcast dari halaman utama? [Y/N]"
+    DowngradeNote   = "Disarankan untuk memblokir karena ada penurunan versi Spotify"
+    UpdBlock        = "Pembaruan Spotify diblokir"
+    UpdUnblock      = "Pembaruan Spotify tidak diblokir"
+    UpdSelect       = "Ingin memblokir pembaruan Spotify? [Y/N]"
+    CacheOn         = "Membersihkan cache diaktifkan ({0})"
+    CacheOff        = "Membersihkan cache tidak diaktifkan"
+    CacheSelect     = "Ingin mengatur pembersihan cache otomatis? [Y/T]"
+    CacheDays       = "Cache yang lebih lama: XX hari untuk dibersihkan "
+    CacheDays2      = "Masukkan jumlah hari dari 1 hingga 100"
+    NoVariable      = "Tidak menemukan variabel"
+    NoVariable2     = "di xpui.js"
+    NoVariable3     = "di licenses.html"
+    NoVariable4     = "di html"
+    NoVariable5     = "di home-v2.js"
+    NoVariable6     = "di xpui-desktop-modals.js"
+    ModSpoti        = "Menambal Spotify..."
+    Error           = "Kesalahan"
+    FileLocBroken   = "Lokasi file Spotify rusak, hapus instalan klien dan jalankan skrip lagi"
+    Spicetify       = "Spicetify terdeteksi"
+    NoRestore       = "SpotX telah diinstal, tetapi file untuk memulihkan xpui.js.bak dan xpui.css.bak tidak ditemukan. `nHarap hapus instalan klien Spotify dan jalankan Install.bat lagi"
+    ExpSpotify      = "Fitur eksperimental yang dioperasikan oleh Spotify"
+    ExpStandart     = "Fitur eksperimental SpotX tidak termasuk"
+    NoRestore2      = "SpotX sudah terinstal, xpui.bak tidak ditemukan. Harap hapus instalan klien Spotify dan jalankan Install.bat lagi"

```

Comment:
Lost `n

Reply:
@amd64fox isn't "`n" just not there?
only on line 53